docs/sphinx/sections/api/apidocs/libraries/dagster-aws.rst

```diff
@@ -38,6 +32,17 @@ S3
 .. autoconfigurable:: dagster_aws.s3.s3_pickle_io_manager
   :annotation: IOManagerDefinition
 
+File Manager (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File Manager APIs will be removed from the main dagster library in version ``1.1.0``, and these APIs will be removed in version ``0.17.0`` of ``dagster-aws``.
```

Comment:
I'm not confident that we should remove file managers.  I think it's worth thinking about them a little more before making this promise.

Reply:
Roger. Right now, FileManager is only accessible via a submodule import. Should we change that?

Reply:
Interesting - eh no strong opinion either way, but if you left it up to me, I'd say leave it how it is